public/context.json

```diff
@@ -35,7 +36,10 @@
     "title": {"@id": "dcterms:title"},
     "abstract": {"@id": "dcterms:abstract"},
 
+
     "edition": {"@id": "bf:editionStatement"},
+    "projection": {"@id": "bf:cartographicProjection"},
+    "scale": {"@id": "bf:cartographicScale"},
```

Comment:
Oh, I think the issue is that those predicates are defined in Bibframe 1 (http://bibframe.org/vocab/) — not Bibframe 2 (http://id.loc.gov/ontologies/bibframe/).

So I think you just need to define a `bf1` or whatever namespace for the Bibframe 1 predicates, and then that should be fine.